app/assets/javascripts/components/utils.js.coffee

```diff
@@ -132,3 +132,9 @@ class @Utils
       .fail (xhr, status, error) ->
         alert('Error: ' + error)
         callback()
+
+  @select2TagClickHandler: (e, elem) ->
+    if e.which == 1
```

Comment:
It'd just suck to edit your options, then try to click on an Agent tag, but accidentally click the link and lose your work.  I'm okay with this if you like it, but I'd prefer a confirm dialog for this case.  I don't think we have to add it everywhere, we could start by just adding it here

```coffee
@select2TagClickHandler: (e, elem) ->
  if confirm('Are you sure you want to navigate away?  Any changes will be lost.')
    if e.which == 1
      ...
```